automated install/basic-install.sh

```diff
@@ -2638,6 +2662,8 @@ main() {
         fi
     fi
 
+    echo "password is " ${pw}
```

Comment:
You need to use double quotes around the variable here.

```suggestion
    echo "password is " "${pw}"
```

or

```suggestion
    echo "password is ${pw}"
```